src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,28 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+
+    private static final int MIN_REQUIRE_AGE = 18;
+    private static final int MIN_REQUIRE_LENGTH_PASS = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new NullPointerException("User is null");
```

Comment:
```suggestion
        if (user == null) {
            throw new RuntimeException("User is null");
```

It better no to throw NPE